raiden-ts/src/logger.ts

```diff
@@ -7,9 +7,9 @@ Object.defineProperty(logging, 'methodFactory', {
   get() {
     return curMethodFactory;
   },
-  set(func: typeof curMethodFactory) {
+  set(func: typeof curMethodFactory & { allow_overwrite?: boolean }) {
```

Comment:
Nitpick, can we name this var in camelCase instead of snake_case? To keep consistency with the our naming convention

Reply:
Ofc. Sorry, in my free time I currently use languages where snake case is the standard. 🙈

Reply:
✔️